### DISCUSSION TITLE
How to use Ticket vendor?

### DISCUSSION THREAD

**Discussion by Scwestbrook** [ChAoSUnItY/EkiLib#3] [Q&A] [answered]
How can I obtain a ticket from the vendor?

**Comment by ChAoSUnItY** [comment, marked as answer]
To obtain a ticket from vendor, right click on the vendor would open a screen, type numbers in the text box (It's at the center of screen), then click confirm, it'll return a ticket in the slot of the vendor (or pop to the world if the slot is full).

**Reply by Scwestbrook** [reply]
Thanks!!!

**Reply by Scwestbrook** [reply]
How do I rename a station sign?

**Reply by Scwestbrook** [reply]
Also, what's the IC ticket for?

**Reply by ChAoSUnItY** [reply]
1. Open your station tab and you can click the stations in the center of the tab, after you clicked and selected, press the `create/ modify station` button. The screen would allow you to edit the station you just selected.
2. IC ticket is still not implemented into mod. Currently it has no usage.

**Reply by Scwestbrook** [reply]
Thank you!!!
